mmv1/products/developerconnect/Connection.yaml

```diff
@@ -45,13 +45,24 @@ async:
     message: 'message'
 custom_code:
 examples:
-  - name: 'developer_connect_connection_basic'
+  - name: 'developer_connect_connection_new'
```

Comment:
this looks good but we should keep the basic example still, you seem to have replaced it with your new example but didn't delete the basic tfconfig file.

So in the end we should have `[basic, new, developer_connect_connection_existing_credentials]

Reply:
The old basic example became the `existing_credentials` example. So it's really two tests [new, existing_credentials] + docs